### ISSUE TITLE
All bookmarks in folders breaks bookmark page generation

### ISSUE THREAD

**Issue by Wjxfi** [Slion/Fulguris#493]
# Description
changes in the bookmark manager do not appear in the basic new tab "bookmarks". Namely, new folders and placing bookmarks in them.

# Environment
 - Device: poco x3 pro
 - OS: android 13, miui 13.0.5
 - Version: 1.8.28

# To reproduce
Steps to reproduce the behaviour:
1. go to bookmark management
2. change a lot of things, put all your bookmarks in different folders.
3. create a new tab, default: bookmarks
4. bookmarks have not changed, they have not become organized in new folders created in the bookmark manager

# Expected behaviour
Bookmarks has changed, in a new tab, "bookmarks"

# Screenshots
![d33f29cd-d7f3-402a-9f1c-57c6661dd283.jpg](https://github.com/Slion/Fulguris/assets/113500363/5743005d-099c-4928-a1a7-e213ba704ac8)

![4d8987e6-237f-4723-b8ce-5735c42cfcc6.jpg](https://github.com/Slion/Fulguris/assets/113500363/d721b492-8c9a-4735-bc00-90613afe7089)

# Additional context
It's a serious and very frustrating problem for me.

Edited: after this action, bookmarks are not changed, added, or updated on the bookmarks home page.

**Comment by Slion** [comment]
It does not work if you refresh the tab?
Does it work of you restart the app?

**Comment by Slion** [comment]
Only happens if you only have folders at the root and no bookmark. Just make sure you have at least one bookmark at the root to workaround that issue.

**Comment by Slion** [comment]
Yes indeed, it should be fixed. Could be a while until I get around it though.
